### PR TITLE
Update code style

### PR DESCRIPTION
**Pull request by cthoyt**
@AldisiRana please read through and make sure it doesn't look like I've introduced any bugs

### CODE REVIEW

File: src/seffnet/chemical_similarities.py [hunk 1]
```diff
@@ -120,65 +124,74 @@ def create_clustered_chemsim_graph(
...
     :param mapping_file: an existing dataframe with pubchemIDs and Smiles
     """
     if not rebuild and weighted and os.path.exists(DEFAULT_CHEMSIM_WEIGHTED_PICKLE):
         return nx.read_edgelist(DEFAULT_CHEMSIM_WEIGHTED_PICKLE)
-    elif not rebuild and os.path.exists(DEFAULT_CHEMSIM_PICKLE):
+    elif not rebuild and not weighted and os.path.exists(DEFAULT_CHEMSIM_PICKLE):
         return nx.read_edgelist(DEFAULT_CHEMSIM_PICKLE)
-    if type(fullgraph) == pybel.struct.graph.BELGraph:
+
+    if isinstance(fullgraph, BELGraph):
```
Comment (cthoyt) [inline src/seffnet/chemical_similarities.py:188]: note that `isinstance(x, y)` is safer than `type(x) == y`

File: src/seffnet/chemical_similarities.py [hunk 2]
```diff
@@ -190,42 +203,44 @@ def get_similarity_graph(
 
     pubchem_id_to_fingerprint = get_fingerprints(pubchem_id_to_smiles)
 
-    chemsim_graph = pybel.BELGraph(name, version, description, authors, contact)
+    chemsim_graph = pybel.BELGraph(name=name, version=version)
```
Comment (cthoyt) [inline src/seffnet/chemical_similarities.py:206]: these variables were never used

File: src/seffnet/chemical_similarities.py [hunk 3]
```diff
@@ -120,65 +124,74 @@ def create_clustered_chemsim_graph(
...
-    if type(fullgraph) == pybel.struct.graph.BELGraph:
+
+    if isinstance(fullgraph, BELGraph):
         fullgraph_without_chemsim = fullgraph
     else:
         fullgraph_without_chemsim = pybel.from_pickle(fullgraph)
+
     pubchem_ids = []
-    for node in fullgraph_without_chemsim.nodes():
+    for node in fullgraph_without_chemsim:
```
Comment (cthoyt) [inline src/seffnet/chemical_similarities.py:194]: graphs are iterable over their nodes

File: src/seffnet/chemical_similarities.py [hunk 4]
```diff
@@ -234,60 +249,67 @@ def get_combined_graph_similarity(
...
+
     relabel_graph = {}
-    for ind, row in mapping_df.iterrows():
-        if row['namespace'] == PUBCHEM_NAMESPACE:
-            relabel_graph[pybel.dsl.Abundance(namespace=PUBCHEM_NAMESPACE, identifier=row['identifier'])] = \
-                row['node_id']
-        elif row['namespace'] == UNIPROT_NAMESPACE:
-            relabel_graph[pybel.dsl.Protein(namespace=UNIPROT_NAMESPACE, identifier=row['identifier'],
-                                            name=row['name'])] = row['node_id']
+    for node_id, namespace, identifier, name in mapping_df[['node_id', 'namespace', 'identifier', 'name']].values:
```
Comment (cthoyt) [inline src/seffnet/chemical_similarities.py:274]: using `df.values` eliminates the index, and slicing it allows us to explicitly take the variables out of the row we're using and assign them

**Review by AldisiRana — APPROVED** [review]

**Pull request merged**